.mlc.toml

```diff
@@ -5,3 +5,5 @@ ignore-links=["../../*", "/docs/*",
 ]
 # Path to the root folder used to resolve all relative paths
 root-dir="./docs"
+offline = true
```

Comment:
But checking external links is also valuable. It can warn us if a resource we refer to is no longer present. Is it possible to retry on broken links instead of completely cutting them off?